### PR TITLE
test(ui): Fix flaky tasks and dashboard tests

### PR DESCRIPTION
**Pull request by hoorayimhelping**
Fixes the flaky nature of our e2e tests so they're more deterministic.

- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass

### CODE REVIEW

File: ui/cypress/e2e/tasks.test.ts [hunk 1]
```diff
@@ -61,80 +61,64 @@ from(bucket: "${name}")
...
-
-      cy.getByTestID('task-card').should('have.length', 2)
-
-      cy.getByTestID('task-card')
-        .first()
-        .trigger('mouseover')
-        .within(() => {
-          cy.getByTestID('context-delete-menu').click()
-          cy.getByTestID('context-delete-task').click()
+  describe('When tasks already exsist', () => {
```
Comment (ebb-tide) [inline ui/cypress/e2e/tasks.test.ts]: exists?

Comment (hoorayimhelping) [reply ui/cypress/e2e/tasks.test.ts]: lol @zoesteinkamp ☝️  😂 

good catch @ebb-tide ! we had a little laugh about Zoe's spelling abilities during our pairing session

File: ui/cypress/e2e/tasks.test.ts [hunk 2]
```diff
@@ -61,80 +61,69 @@ from(bucket: "${name}")
...
+    it('can edit a task', () => {
+      // Disabling the test
+      cy.getByTestID('task-card--slide-toggle').should('have.class', 'active')
 
-    cy.getByTestID('task-card--slide-toggle').click()
+      cy.getByTestID('task-card--slide-toggle').click()
 
-    cy.getByTestID('task-card--slide-toggle').should('not.have.class', 'active')
-  })
+      cy.getByTestID('task-card--slide-toggle').should(
```
Comment (Asalem1) [inline ui/cypress/e2e/tasks.test.ts]: linter might want this on 1 line

File: ui/cypress/e2e/tasks.test.ts [hunk 3]
```diff
@@ -61,80 +61,69 @@ from(bucket: "${name}")
...
-        .type('{enter}')
+      cy.getByTestID('notification-success').should('exist')
+      cy.getByTestID('task-card').should('contain', newName)
     })
 
-    cy.fixture('routes').then(({orgs}) => {
-      cy.get('@org').then(({id}: Organization) => {
-        cy.visit(`${orgs}/${id}/tasks`)
-      })
+    it.only('can delete a task', () => {
```
Comment (Asalem1) [inline ui/cypress/e2e/tasks.test.ts]: `.only`?

File: ui/cypress/e2e/tasks.test.ts (hunk 2 above)
Comment (hoorayimhelping) [reply ui/cypress/e2e/tasks.test.ts]: this was produced by running `yarn prettier:fix` /shrug

Comment (Asalem1) [reply ui/cypress/e2e/tasks.test.ts]: 🤷‍♂

File: ui/cypress/e2e/tasks.test.ts (hunk 3 above)
Comment (hoorayimhelping) [reply ui/cypress/e2e/tasks.test.ts]: hot tip lads and lassies: `yarn prettier:fix` doesn't fix `it.only`.

Comment (Asalem1) [reply ui/cypress/e2e/tasks.test.ts]: 🤣

**Pull request merged**